erpnext/public/js/controllers/accounts.js

```diff
@@ -11,6 +11,7 @@ frappe.ui.form.on(cur_frm.doctype, {
 		$(frm.wrapper).on('grid-row-render', function(e, grid_row) {
 			if(in_list(['Sales Taxes and Charges', 'Purchase Taxes and Charges'], grid_row.doc.doctype)) {
 				erpnext.taxes.set_conditional_mandatory_rate_or_amount(grid_row);
+				erpnext.taxes.update_category_and_add_deduct_tax(grid_row);
```

Comment:
Don't call this on setup, it's throwing an error on the console trying to refresh the table